testsuite/forge.py

```diff
@@ -448,14 +448,22 @@ def set_output(self, output: str) -> None:
     def set_debugging_output(self, output: str) -> None:
         self.debugging_output = output
 
-    def format(self) -> str:
-        output = "\n".join([
+    def format(self, context: ForgeContext) -> str:
+        dashboard_link = get_dashboard_link(
+            context.forge_cluster_name,
+            context.forge_namespace,
+            context.forge_chain_name,
+            (self.start_time, self.end_time),
+        )
+        output_lines = []
+        if not self.succeeded():
+            output_lines.append(self.debugging_output)
+        output_lines.extend([
             f"Forge output: {self.output}",
             f"Forge {self.state.value.lower()}ed",
+            f"Dashboard Link: {dashboard_link}",
```

Comment:
Printing a link to the logs would be awesome too 😄

Reply:
I'm just gonna print the pre / post like on forge github action